.editorconfig

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
```

Comment:
moderate: Not sure if the base commit is copyrighted to CERN (if so it might be the author of the commit should be `Invenio <info@inveniosoftware.org>`. Otherwise it might be your institution... @lnielsen is the copyrights expert :).